src/content_sao.cpp

```diff
@@ -920,21 +918,10 @@ std::string PlayerSAO::getClientInitializationData(u16 protocol_version)
 	return os.str();
 }
 
-std::string PlayerSAO::getStaticData()
+void PlayerSAO::getStaticData(std::string *result) const
 {
 	FATAL_ERROR("Deprecated function (?)");
```

Comment:
Macro FATAL_ERROR already set FUNCTION_NAME, but i will remove the (?) and the return
```
  fatal_error_fn((msg), __FILE__, __LINE__, FUNCTION_NAME)
```